### ISSUE TITLE
Incompatibility with Werzeug 3.0.0 (Flask 3.0.0)

### ISSUE THREAD

**Issue by TheoLechemia** [maxcountryman/flask-login#799]
**Describe the bug**
Just import `LoginManager` from `flask_login` in your code, with Werkzeug 3.0.0 (given version with Flask 3.0.0)

```
    from flask_login import LoginManager
/opt/hostedtoolcache/Python/3.9.18/x64/lib/python3.9/site-packages/flask_login/__init__.py:12: in <module>
    from .login_manager import LoginManager
/opt/hostedtoolcache/Python/3.9.18/x64/lib/python3.9/site-packages/flask_login/login_manager.py:33: in <module>
    from .utils import _create_identifier
/opt/hostedtoolcache/Python/3.9.18/x64/lib/python3.9/site-packages/flask_login/utils.py:14: in <module>
    from werkzeug.urls import url_decode
E   ImportError: cannot import name 'url_decode' from 'werkzeug.urls' (/opt/hostedtoolcache/Python/3.9.18/x64/lib/python3.9/site-packages/werkzeug/urls.py)
```

**Desktop (please complete the following information):**
 - OS: all
 - Browser all
 - Version 0.6.2

In fact, `url_encode` is deprecated from Werkzeug since `2.3`. They suggest to use ` urllib.parse.urlencode` instead.
See : https://werkzeug.palletsprojects.com/en/2.3.x/urls/#werkzeug.urls.url_encode

I can provide a PR if needed ?

**Comment by davidism** [comment]
Please search for duplicates first.

**Comment by davidism** [comment]
Duplicate of #744, fixed by #746. A volunteer maintainer will work on making a release when they have time available.
